javascript/javascript1/week1/dogage.js

```diff
@@ -0,0 +1,16 @@
+let dogYearOfBirth= 2017; 
+let dogYearFuture = 2027;
+ dogYear = dogYearFuture - dogYearOfBirth;
+ console.log(dogYear);
+  shouldShowResultInDogYears = dogYear * 7 ; 
+  console.log(shouldShowResultInDogYears);
+
+ if (shouldShowResultInDogYears == true){ 
```

Comment:
This condition will always return false because you compare   shouldShowResultInDogYears  is number. so you should compare it with a number